packages/common/test/core/schemas/internal/getEditorSchemas.test.ts

```diff
@@ -143,7 +142,6 @@ describe("getEditorSchemas: ", () => {
 
     { type: GroupEditorTypes[0], module: GroupBuildEditUiSchema },
     { type: GroupEditorTypes[1], module: GroupBuildSettingsUiSchema },
-    { type: GroupEditorTypes[2], module: GroupBuildDiscussionsUiSchema },
```

Comment:
Rather than removing, this should be updated to something like:

```js
    { type: GroupEditorTypes[2], module: EntityBuildDiscussionSettingsUiSchema },
```